src/mpyl/cli/plan.py

```diff
@@ -61,16 +62,25 @@ def plan(ctx, config, properties):
     )
 
 
-@plan.command("create")
+@plan.command("discover")
+@click.option(
+    "--project",
+    "-p",
+    type=click.STRING,
+    required=False,
+    help="Limit the run plan to only this project",
```

Comment:
Is the manual sub-select limited to only a single project on purpose? Should this not be expanded to a list?

Reply:
Yes, currently we only support manually selecting a single project, so we ported that 1:1 here. If teams ask for it we can consider supporting multiple projects.